settings/DevHome.Settings/ViewModels/ExtensionSettingsViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using AdaptiveCards.Rendering.WinUI3;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.WinUI;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Common.Views;
+using DevHome.Logging;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.ApplicationModel;
+
+namespace DevHome.Settings.ViewModels;
+
+public partial class ExtensionSettingsViewModel : ObservableObject
+{
+    private readonly IPluginService _pluginService;
+
+    public ExtensionSettingsViewModel(IPluginService pluginService)
+    {
+        _pluginService = pluginService;
+
+        Breadcrumbs = new ObservableCollection<Breadcrumb> { };
+    }
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs
+    {
+        get; set;
+    }
+
+    [RelayCommand]
+    private async Task OnSettingsContentLoadedAsync(PluginAdaptiveCardPanel pluginAdaptiveCardPanel)
+    {
+        var pluginWrappers = await _pluginService.GetInstalledPluginsAsync(true);
+
+        var navigationService = Application.Current.GetService<INavigationService>();

```

Comment:
nit: INavigationService can be injected similar to how IPluginService is done 😉

Reply:
Fixed